com.unity.render-pipelines.universal/Editor/ForwardRendererDataEditor.cs

```diff
@@ -5,129 +5,7 @@
-                EditorGUI.indentLevel--;
-            }
-            EditorGUI.indentLevel--;
-            EditorGUILayout.Space();
-#endif
-            EditorGUILayout.LabelField("Shadows", EditorStyles.boldLabel);
-            EditorGUI.indentLevel++;
-            EditorGUILayout.PropertyField(m_ShadowTransparentReceiveProp, Styles.shadowTransparentReceiveLabel);
-            EditorGUI.indentLevel--;
-            EditorGUILayout.Space();
-
-            EditorGUILayout.LabelField("Overrides", EditorStyles.boldLabel);
-            EditorGUI.indentLevel++;
-            EditorGUILayout.PropertyField(m_DefaultStencilState, Styles.defaultStencilStateLabel, true);
-            SerializedProperty overrideStencil = m_DefaultStencilState.FindPropertyRelative("overrideStencilState");
-#if ENABLE_RENDERING_PATH_UI
-            if (overrideStencil.boolValue && m_RenderingMode.intValue == (int)RenderingMode.Deferred)
-            {
-                CompareFunction stencilFunction = (CompareFunction)m_DefaultStencilState.FindPropertyRelative("stencilCompareFunction").enumValueIndex;
-                StencilOp stencilPass = (StencilOp)m_DefaultStencilState.FindPropertyRelative("passOperation").enumValueIndex;
-                StencilOp stencilFail = (StencilOp)m_DefaultStencilState.FindPropertyRelative("failOperation").enumValueIndex;
-                StencilOp stencilZFail = (StencilOp)m_DefaultStencilState.FindPropertyRelative("zFailOperation").enumValueIndex;
-                bool invalidFunction = stencilFunction == CompareFunction.Disabled || stencilFunction == CompareFunction.Never;
-                bool invalidOp = stencilPass != StencilOp.Replace && stencilFail != StencilOp.Replace && stencilZFail != StencilOp.Replace;
-
-                if (invalidFunction || invalidOp)
-                    EditorGUILayout.HelpBox(Styles.invalidStencilOverride.text, MessageType.Error, true);
-            }
-#endif
-            EditorGUI.indentLevel--;
-            EditorGUILayout.Space();
-
-            serializedObject.ApplyModifiedProperties();
-
-            base.OnInspectorGUI(); // Draw the base UI, contains ScriptableRenderFeatures list
-
-            // Add a "Reload All" button in inspector when we are in developer's mode
-            if (EditorPrefs.GetBool("DeveloperMode"))
-            {
-                EditorGUILayout.Space();
-                EditorGUILayout.PropertyField(m_Shaders, true);
-
-                if (GUILayout.Button("Reload All"))
-                {
-                    var resources = target as ForwardRendererData;
-                    resources.shaders = null;
-                    ResourceReloader.ReloadAllNullIn(target, UniversalRenderPipelineAsset.packagePath);
-                }
-            }
-        }
-    }
+    [System.Obsolete("ForwardRendererDataEditor has been deprecated. Use StandardRendererDataEditor instead (UnityUpgradable) -> StandardRendererDataEditor", true)]
```

Comment:
Suggestion, move this to Editor/Deprecated.cs file so we keep all deprecated things in one place and it's easier to remove later.

Reply:
Moved.